src/wp-admin/site-editor.php

```diff
@@ -80,6 +80,44 @@ static function ( $classes ) {
 	}
 }
 
+if ( isset( $_GET['postId'] ) ) {
```

Comment:
Avoids `Warning: Undefined array key "postType"`

```suggestion
if ( isset( $_GET['postId'] ) && isset( $_GET['postType'] ) ) {
```